lib/db/schema/patch-078-079.sql

```diff
@@ -0,0 +1,54 @@
+SET NAMES utf8mb4 COLLATE utf8mb4_bin;
+
+ALTER TABLE recoveryCodes MODIFY COLUMN codeHash BINARY(32);
+ALTER TABLE recoveryCodes ADD COLUMN salt BINARY(32);
```

Comment:
Should these two have the whole `ALGORITHM = INPLACE, LOCK = NONE` malarkey going on?